<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaConsultas</class>
 <widget class="QMainWindow" name="ventanaConsultas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>601</width>
      <height>601</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="consu_apellido">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="consu_continente">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Continentes</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="consu_creencia">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Creencias</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="consu_profesion">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Profesiones</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>37</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consultas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="consu_paises">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Paises</string>
    </property>
   </widget>
   <widget class="QComboBox" name="profesiones_comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>410</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="paises_comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="creencias_comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="continentes_comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="apellidos_comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
